diesel/src/query_dsl/order_dsl.rs

```diff
@@ -12,6 +12,30 @@ use query_source::QuerySource;
 /// columns.
 ///
 /// This is automatically implemented for the various query builder types.
+///
+/// # Examples
+///
+/// ```rust
+/// # #[macro_use] extern crate diesel;
+/// # include!("src/doctest_setup.rs");
+/// #
+/// # table! {
+/// #     users {
+/// #         id -> Integer,
+/// #         name -> VarChar,
+/// #     }
+/// # }
+/// #
+/// # fn main() {
+/// let connection = establish_connection();
+/// #     connection.execute("DELETE FROM users").unwrap();
+/// connection.execute("INSERT INTO users (name) VALUES ('Saul'), ('Steve'), ('Stan')").unwrap();
+/// use self::users::dsl::{users, id, name};
+/// // load all users' names, ordered by their name descending
+/// let ordered_names = users.select(name).order(name.desc()).load(&connection).unwrap();
+/// assert_eq!(Ok(vec![String::from("Steve"), String::from("Stan"), String::from("Saul")]), ordered_names);
```

Comment:
No worries! We're grateful for each contribution :) If it's only minor things (like an `unwrap`), I can fix that before merging